### ISSUE TITLE
JES-explorer is not able to simultaneously show contents of similarly named SYSOUT DDs.

### ISSUE THREAD

**Issue by FreekCornelis** [zowe/zlux#399]
Situation: Zowe 1.8.0, job with 2 STEPs specifying similarly named SYSOUT DD, e.g.:
![image](https://user-images.githubusercontent.com/55885539/75031836-735d7e00-54a7-11ea-86e4-63b1f01e7810.png)

When job J006019 has run:
![image](https://user-images.githubusercontent.com/55885539/75031906-9851f100-54a7-11ea-86df-428b956b1d6e.png)

 it is displayed in the JES-explorer as:
![image](https://user-images.githubusercontent.com/55885539/75032002-d0593400-54a7-11ea-96b8-bdb6e0475e26.png)

Clicking on the top STDOUT file correctly shows the contents:
![image](https://user-images.githubusercontent.com/55885539/75032040-e4049a80-54a7-11ea-9e5c-2dcce57d125f.png)

However, then clicking on the bottom STDOUT file does not open a new tab, nor changes the contents of the already open tab. Only after closing the already open STDOUT tab and clicking again on the bottom STDOUT file its contents are shown:
![image](https://user-images.githubusercontent.com/55885539/75032086-fe3e7880-54a7-11ea-99de-30624d48224a.png)

Then clicking  on the top STDOUT again does not open a new tab for the contents.

In short, JES-explorer is not able to simultaneously show contents of similarly named SYSOUT DDs.
